examples/rust/file_transfer/.rustfmt.toml

```diff
@@ -0,0 +1 @@
+max_width = 120
```

Comment:
The get_started crate does this too because it renders a bit nicer in the guides where the code is automatically copied. Let's leave it in here in this example too.